webapp/utils/tunnel.ts

```diff
@@ -59,3 +61,63 @@ export const getOperationFromDeposit = function (
 
   return btcDepositStatusToAction[deposit.status]
 }
+
+const btcDepositCompletedActions = [
+  BtcDepositStatus.BTC_DEPOSITED,
+  BtcDepositStatus.DEPOSIT_TX_FAILED,
+]
+
+const btcWithdrawCompletedActions = [
+  BtcWithdrawStatus.WITHDRAWAL_FAILED,
+  /*
+  TODO Add the following status when available
+  BtcWithdrawStatus.WITHDRAWAL_SUCCEEDED,
+  BtcWithdrawStatus.WITHDRAWAL_CHALLENGED,
+  BtcWithdrawStatus.CHALLENGE_FAILED
+  */
+]
```

Comment:
we'll have to coordinate to merge this, together with https://github.com/hemilabs/ui-monorepo/pull/742/files#diff-3ec11bc3556e3ad32294f516514bbdf455b6584188edb0e8dae9ef097a83012fR64 (same idea)

Reply:
let's leave like that for now, ok?

Reply:
added